### PR TITLE
Adding info on when/how to use noupdate attribute in xml data files

### PR DESCRIPTION
**Pull request by jcdrubay**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -251,7 +251,9 @@ When declaring a record in XML:
 * Try to group the records by model. In case of dependencies between
   action/menu/views, the convention may not be applicable.
 * Use naming convention defined at the next point
-* The tag `<data>` is only used to set not-updatable data with `noupdate=1`
+* The tag `<data>` is only used to set not-updatable data with `noupdate=1` when your data file contains a mix of "noupdate" data, otherwise, you should use one of this:
```
Comment (lasley) [inline CONTRIBUTING.md]: I think this should be two sentences: `data. Otherwise,`

Also:
 s/one of this/one of these

**Comment by jcdrubay** [comment]
@lasley Where is the bot? I did not find any mention in https://odoo-community.org/page/cla I will take care of the printing and signing now?

BTW, do you know when we can shop 2017 membership? https://odoo-community.org/shop

**Comment by lasley** [comment]
> Where is the bot?

 https://github.com/OCA/maintainer-tools/pull/255#issuecomment-271125777

>  do you know when we can shop 2017 membership

Hmmm got me. @pedrobaeza?

**Comment by jcdrubay** [comment]
@lasley:
 - comments taken into account
 - I just sent the signed ICLA by mail as explained  in https://odoo-community.org/page/cla

@pedrobaeza I can't wait to be a member. It will also enable to start the discussions to launch l10n_vietnam as the community in Vietnam is forming up :)

**Comment by leemannd** [comment]
@jcdrubay Thank you.

**Comment by pedrobaeza** [comment]
@lasley @jcdrubay The January OCA board meeting is tomorrow and we will talk about the 2017 membership. I'll tell you back.

**Comment by pedrobaeza** [comment]
Is the CLA sent?

**Comment by jcdrubay** [comment]
@pedrobaeza Yes it is sent, I just forwarded the email to you.

**Pull request merged**